pom.xml

```diff
@@ -103,6 +104,22 @@
                     <attach>false</attach>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <release>11</release>
```

Comment:
we must compile for Java 8 still (which is not a problem for JDK11, though)

Reply:
Also "release" is only available since JDK9 and we must make sure that compilation still works with JDK8.